src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj

```diff
@@ -28,12 +28,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <PackageLicenseFile>LICENSE.md</PackageLicenseFile>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
Not needed. Mistakenly `PackageLicenseFile` was added twice.

Reply:
Done